pyaedt/desktop.py

```diff
@@ -1605,7 +1604,34 @@ def install(package_path, package_name=None):
                 break
         if not full_path:
             raise FileNotFoundError("Error finding the package.")
-        product = toolkit["installation_path"]
+        self.add_script_to_menu(
+            toolkit_name=toolkit_name, script_path=full_path, script_image=toolkit, product=toolkit["installation_path"]
+        )
+
+    @pyaedt_function_handler()
+    def add_script_to_menu(self, toolkit_name, script_path, script_image=None, product="Project"):  # pragma: no cover
+        """Add a script to Ribbon menu. This functionality is available from AEDT 2023 R2.
+        PyAEDT must be installed in AEDT to allow this functionality to run.
+        Please refer to ``"https://aedt.docs.pyansys.com/version/stable/Getting_started/Installation.html"``.
+
+        Parameters
+        ----------
+        toolkit_name : str
+            Name of the toolkit as it will appear in AEDT.
+        script_path : str
+            Full path to the script file.
+        script_image : str, optional
+            Full path to the image logo (30x30 pixels png) to be added into the UI.
+        product : str, optional
+            Product to which the toolkit will apply. It can be Project to apply to all designs or specific (eg. HFSS).
```

Comment:
```suggestion
            Product to which the toolkit applies. The default is ``"Project"``, in which case it
            applies to all designs. You can also specify a product, such as ``"HFSS"``.
```
Would you use lowercase for ``"HFSS"``.